templates/raspbian_cattlepi/resources/usr/share/initramfs-tools/scripts/cattlepi-base/helpers

```diff
@@ -355,3 +355,60 @@ cattlepi_build_root_filesystem()
     echo $CATTLEPI_BASE > "${rootmnt}/cattlepi/base"
     echo "boot/${CATTLEPI_ID}/config" > "${rootmnt}/cattlepi/base_relative_config"
 }
+
+cattlepi_bring_up_wlan0() {
+    interface="wlan0"
+    wait_limit="30"
+    online=false
+
+    echo "Bringing up wlan0"
+
+    if [ ! -x "/sbin/wpa_supplicant" ]; then
```

Comment:
I think the file can exist and this fail (not executable?)

Reply:
You're right.  Will update.

Reply:
Commented on this without looking (it was early this morning).

`-x` should ensure that the file exists and is executable.